addons/website_event_online/controllers/main.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_event.controllers.main import WebsiteEventController
+from odoo.http import request
+
+
+class WebsiteEventSaleController(WebsiteEventController):
+
+    def _create_attendees_from_registration_post(self, event, registration_data):
```

Comment:
SideNote: something we could do is propagate partner_id to visitor when updating registrations. Indeed if I set the partner on the SO / registration, we could propagate it to the visitor and fill missing informations is we have some fields not set.